app/assets/javascripts/guiders.js

```diff
@@ -0,0 +1,42 @@
+$(document).on('turbolinks:load', function() {
+  $('.js-go-link').on('click', function() {
```

Comment:
@andrewvos cool, might be worth holding fire on the JS commentary as @guntrisoft is giving it a full going over tomorrow.